symbolic-debuginfo/src/object.rs

```diff
@@ -60,40 +60,39 @@ macro_rules! map_result {
 
 /// An error when dealing with any kind of [`Object`](enum.Object.html).
 #[non_exhaustive]
-#[allow(clippy::large_enum_variant)]
-#[derive(Debug, Fail)]
+#[derive(Debug, Error)]
```

Comment:
On most errors we started deriving clone, except on this one. I'm considering to remove all `Clone` implementations, but let's check if we need them in Symbolicator.

Reply:
note to self: remove `Clone` on all the other custom error types…